docs/intro/tutorial05.txt

```diff
@@ -285,7 +285,7 @@ more comprehensively:
         was_published_recently() should return False for questions whose
         pub_date is older than 1 day.
         """
-        time = timezone.now() - datetime.timedelta(days=30)
+        time = timezone.now() - datetime.timedelta(days=1)
```

Comment:
Would it make sense to change the next test to use `datetime.timedelta(hours=23, minutes=59, seconds=59)` instead of `datetime.timedelta(hours=1)` so that the boundary is tested on both sides?